### PR TITLE
report-flow-ustc:1.1.0, touying-flow:1.1.0 and touying-pres-ustc:0.2.0

### PR DESCRIPTION
**Pull request by Quaternijkon**
I am submitting
- [ ] a new package
- [x] an update for a package

Description: 
- Add interface to facilitate user modifications of resource styles, making the template more versatile. 
- Adjust some styles to create a more harmonious design. 
- Simplify the initialization code.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE
- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/touying-pres-ustc/0.2.0/content/example/main.typ [hunk 1]
```diff
@@ -0,0 +1,403 @@
+#import "@preview/touying-pres:0.2.0": *
```
Comment (elegaanz) [inline packages/preview/touying-pres-ustc/0.2.0/content/example/main.typ:1]:
```suggestion
#import "@preview/touying-pres-ustc:0.2.0": *
```

Comment (Quaternijkon) [reply packages/preview/touying-pres-ustc/0.2.0/content/example/main.typ]: Oops, by coincidence, I had a local copy of this package, so I didn’t realize the mistake. I have corrected it now. Thank you!

File: packages/preview/touying-pres-ustc/0.2.0/content/example/main.typ [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+#let s = (s.methods.numbering)(self: s, section: "1.", "1.1")
+#let s = (s.methods.info)(
+  self: s,
+  title: [Typst template for University of Science and Technology of China],
+  subtitle: [Continuously Improving...],
+  author: [Quaternijkon],
+  date: datetime.today(),
+  institution: [School of Computer Science and Technology, USTC],
+  logo: image("../../assets/img/USTC_logo_side.svg", width: 50%),
+  head-logo: image("../../assets/img/ustc_logo_side.svg",width: 20%),
```
Comment (elegaanz) [inline packages/preview/touying-pres-ustc/0.2.0/content/example/main.typ:13]: These two files can't be accessed from your template. You should either copy these images to `content/example/img` and reference this copy, or export `#let ustc-logo = image("/assets/img/USTC_logo_side.svg")` from the package and use that variable here.

Comment (elegaanz) [reply packages/preview/touying-pres-ustc/0.2.0/content/example/main.typ]: Yes, they can configure the `image` as they wish, replace the file by another one, or even wrap the image in some other element if they want. It is very flexible.

**Comment by elegaanz** [comment]
Thank you for the updates and for taking the review comments into account :)

**Pull request merged**